src/tests/ftest/util/slurm_utils.py

```diff
@@ -49,9 +49,9 @@ def create_partition(log, control, name, hosts, default='yes', max_time='UNLIMIT
         control (NodeSet): slurm control host
         name (str): slurm partition name
         hosts (NodeSet): hosts to include in the partition
-        default (str, optional): _description_. Defaults to 'yes'.
-        max_time (str, optional): _description_. Defaults to 'UNLIMITED'.
-        state (str, optional): _description_. Defaults to 'up'.
+        default (str, optional): whether this partition should be the default. Defaults to 'yes'.
```

Comment:
'yes' as a default?  What about True/False?

Reply:
This comes from `scontrol create`, not us:
https://slurm.schedmd.com/scontrol.html#OPT_Default